templates/launch.html

```diff
@@ -21,12 +22,44 @@ <h1>Launch a survey</h1>
         </select>
     </div>
 
+    <p>----------</p>
+
+    <strong><u>Launch by Schema URL</u></strong>
+    <div class="field-container">
+        <span>
+            <label for="schema-url-survey-type">Survey Type</label>
+            <select name="survey-types" id="schema-url-survey-type">
+                <option selected disabled>Select Survey Type</option>
+                {{ range $surveyType, $schemasList := .Schemas }}
+                <option name="{{ $surveyType }}" value="{{ $surveyType }}" data-schema-url-survey-type="{{ $surveyType }}">{{ $surveyType }}</option>
+            {{ end }}
+            </select>
+        </span>
+    </div>
+
+    <div class="field-container">
+        <span>
+            <label for="schema-url">Schema URL</label>
+            <input id="schema-url" name="survey_url" type="text" class="qa-schema_url">
+            <input type="submit" onClick="return loadMetadataForSchemaUrl()" value="Load Schema" class="qa-btn-submit-dev btn" id="schema-url-btn"/>
+        </span>
+    </div>
+
+    <div class="field-container" id="schema-url-name">
+        <span>
+            <label for="schema-url-name-input">Schema Name</label>
+            <input id="schema-url-name-input" name="schema_name" type="text" class="qa-schema_url_name" disabled>
+        </span>
+    </div>
+
+    <p>----------</p>
+
```

Comment:
```suggestion
    <strong><u>Launch by Schema URL</u></strong>
    <div class="field-container">
        <span>
            <label for="schema-url-survey-type">Survey Type</label>
            <select name="survey-types" id="schema-url-survey-type">
                <option selected disabled>Select Survey Type</option>
                {{ range $surveyType, $schemasList := .Schemas }}
                <option name="{{ $surveyType }}" value="{{ $surveyType }}" data-schema-url-survey-type="{{ $surveyType }}">{{ $surveyType }}</option>
            {{ end }}
            </select>
        </span>
    </div>

    <div class="field-container">
        <span>
            <label for="schema-url">Schema URL</label>
            <input id="schema-url" name="survey_url" type="text" class="qa-schema_url">
        </span>
    </div>

    <div class="field-container" id="schema-url-name">
        <span>
            <label for="schema-url-name-input">Schema Name</label>
            <input id="schema-url-name-input" name="schema_name" type="text" class="qa-schema_url_name" disabled>
        </span>
    </div>
    <div class="field-container">
        <input type="submit" onClick="return loadMetadataForSchemaUrl()" value="Load Schema" class="qa-btn-submit-dev btn" id="schema-url-btn"/>
    </div>
    <p>----------</p>

```

So it's similar to the existing "design".

Reply:
Updated.